### PR TITLE
Dependency updates to latest and supporting code changes

### PR DESCRIPTION
**Pull request by sumesh-aot**
Dependency updates to latest and supporting code changes
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-api/tests/conftest.py [hunk 1]
```diff
@@ -47,85 +49,53 @@ def client_ctx(app):  # pylint: disable=redefined-outer-name
...
-def session(app, db):  # pylint: disable=redefined-outer-name, invalid-name
+def session(app, database):  # pylint: disable=redefined-outer-name, invalid-name
     """Return a function-scoped session."""
     with app.app_context():
-        conn = db.engine.connect()
-        txn = conn.begin()
-
-        options = dict(bind=conn, binds={})
-        sess = db.create_scoped_session(options=options)
+        # TODO, refactor to improve test execution time.
```
Comment (sumesh-aot) [inline forms-flow-api/tests/conftest.py:82]: This is causing delay in test execution, will continue looking for improvements

**Review by abilpraju-aot — APPROVED** [review]
LGTM

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-api/src/formsflow_api/models/application.py [hunk 2]
```diff
@@ -496,7 +495,9 @@ def set_sort(sort_by, sort_order):
         parent_formid_and_sum_of_application = (
             db.session.query(
                 distinct_form_id_count.c.parent_form_id,
-                db.func.sum(distinct_form_id_count.c.application_count).label(  # pylint: disable=not-callable
+                db.func.sum(
+                    distinct_form_id_count.c.application_count
+                ).label(  # pylint: disable=not-callable
```
Comment (sreehari-aot) [inline forms-flow-api/src/formsflow_api/models/application.py:500]: I think the pylint disable can be removed

**Review by sreehari-aot — APPROVED** [review]

**Comment by abhilash-aot** [comment]
There is conflict in the code, need to resolve

**Comment by sumesh-aot** [comment]
Opening a new PR, as I messed up some commits

**Pull request closed without merging**